cms/models.py

```diff
@@ -72,6 +72,43 @@ def save(self, *args, **kwargs):
         super(LearningOutcomesPage, self).save(*args, **kwargs)
 
 
+class WhoShouldEnrollPage(CourseProgramChildPage):
+    """
+    Who should enroll child page for "Who Should Enroll" section.
+    """
+
+    subpage_types = []
+
+    image = models.ForeignKey(
+        Image,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        help_text="Image size must be at least 870x500 pixels.",
+    )
+    content = StreamField(
+        [
+            (
+                "item",
+                blocks.RichTextBlock(
+                    icon="plus", features=["bold", "italic", "ol", "ul"]
+                ),
+            )
+        ],
+        blank=False,
+        help_text='Contents of the "Who Should Enroll" section.',
+    )
+
+    content_panels = [StreamFieldPanel("content"), FieldPanel("image")]
+
+    def save(self, *args, **kwargs):
+        # autogenerate a unique slug so we don't hit a ValidationError
+        self.title = "Who Should Enroll"
+        self.slug = slugify("{}-{}".format(self.get_parent().id, self.title))
+        super().save(*args, **kwargs)
```

Comment:
I prefer: super(WhoShouldEnrollPage, self).save(*args, **kwargs)

Reply:
Actually, I have used that pattern in #142 , so to keep code consistent that's why I was requesting.